src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -48,7 +49,9 @@ public void setUp() {
         JsonAddressBookStorage addressBookStorage =
                 new JsonAddressBookStorage(temporaryFolder.resolve("addressBook.json"));
         JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(temporaryFolder.resolve("userPrefs.json"));
-        StorageManager storage = new StorageManager(addressBookStorage, userPrefsStorage, new RoomOccupancyStorage());
+        RoomOccupancyStorage roomOccupancyStorage = new RoomOccupancyStorage(Paths.get("lol"),
```

Comment:
is the name of the file ``lol``?

Reply:
I think that is probably the temporary files used for testing? 🤣 Shall we rename the test files?